data/json/recipes/armor/feet.json

```diff
@@ -1955,5 +1955,36 @@
     ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
     "tools": [ [ [ "hotcut_any", 1, "LIST" ] ], [ [ "metal_tank", -1 ] ], [ [ "water", -120 ], [ "water_clean", -120 ] ] ]
+  },
+  {
+    "result": "espadrilles",
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 2,
+    "time": "3 h",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 } ],
+    "using": [ [ "tailoring_cotton", 1 ] ],
+    "proficiencies": [ { "proficiency": "prof_basketweaving" } ],
+    "components": [ [ [ "filament", 100, "LIST" ] ] ]
```

Comment:
Ok so the cotton goes on top? the 1000 inches of a single width of thread though seems like it would be too little mass?  do you want to add a foot of rope that makes up the base of each shoe?

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/47b046cd6b6e4f56fc14ba29c769e3b34f42013a/data/json/recipes/armor/feet.json#L2-L15
I'm basically using the cord sandal recipe as a base, this seemed appropriate both in code and as to what the item is representing (an evolution of the cord sandal). 

I assumed the recipe for the cord sandal was correct but if it isn't I can definitely update the espadrilles recipe to use more realistic components and fix the cord sandal recipe in another PR.

Reply:
https://www.youtube.com/watch?v=jmrjjFUuvYQ you use rope to make the soles or you buy premade soles.
https://www.adaspragg.com/2015/08/07/diy-espadrilles/

Reply:
Changed the amount of "filament" needed to craft. "Short rope" is 6 feet of rope and requires 1800 "filament" to craft, 1800/6 = 300, 300 * 2 (one foot for each shoe) = 600 "filament" to craft a pair of Espadrilles. Does that sound good to you?

As for using premade soles I decided to not include this as a crafting option due to the fact that I used the dry plant material in the item definition and the sole items are made of rubber. If you think this is a non issue I'm perfectly fine with adding another recipe.

Better solutions include either making a "rope sole" item, which is out of scope, and updating the crafting system to automatically change the material of the crafted items armor parts to correspond to the material of the components, which is also out of scope to say the least.